boto/pyami/config.py

```diff
@@ -70,6 +68,20 @@ def __init__(self, path=None, fp=None, do_load=True):
                 except IOError:
                     warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
 
+    def __getattr__(self, name):
+        try:
+            impl = self.__dict__['_impl']
+        except KeyError:
+            raise AttributeError(name)
+        return getattr(impl, name)
+
+    def has_option(self, *args, **kwargs):
+        try:
+            impl = self.__dict__['_impl']
```

Comment:
Shouldn't you be able to access `self._impl` directly?  You won't fall back to getattr unless it can't find the attr in the class.